website/src/pages/index.js

```diff
@@ -41,7 +41,7 @@ model_prediction_df = (ONNXModel()
   },
   {
     label: "Model Interpretability",
-    further: "/docs/features/model_interpretability/about",
+    further: "/docs/next/features/responsible_ai/Model%20Interpretation%20on%20Spark",
```

Comment:
This is the only place we will have to remove `/next/` from when Next becomes 0.9.2.

This is the main landing page, and since the `responsible_ai` section is not in 0.9.1 versioned folder, we have to point to the latest one using `docs/next/features/responsible_ai`.